api/proto/teleport/legacy/types/types.proto

```diff
@@ -6337,6 +6339,19 @@ message PluginSCIMSettings {
   string default_role = 2;
 }
 
+// PluginMSTeamsSettings defines the settings for a Microsoft Teams integration plugin
+message PluginMSTeamsSettings {
+  option (gogoproto.equal) = true;
+  // AppId is the Microsoft application ID (uuid, for bots must be underlying app id, not bot's id).
+  string app_id = 1;
+  // TenantId is the Microsoft tenant ID.
+  string tenant_id = 2;
+  // TeamsAppId is the Microsoft teams application ID.
+  string teams_app_id = 3;
+  // Region bot framework api AP region.
+  string region = 4;
+}
```

Comment:
All the other plugins seem to have at least one fallback recipient defined in the configuration in case there is no AMR configured, or the AMR doesn't match. What happens if there's no AMR here? Will the plugin crash because its configuration requires at least one recipient?

As a user, I would expect the integration to work out of the box after the setup flow,. I should not have to setup AMRs to get the notifications sent to a channel or myself.